go.mod

```
@@ -7,7 +7,7 @@ require (
 	github.com/BurntSushi/toml v1.3.2
 	github.com/containers/libtrust v0.0.0-20230121012942-c1716e8a8d01
 	github.com/containers/ocicrypt v1.1.7
-	github.com/containers/storage v1.48.0
+	github.com/containers/storage 4a3a3019d7659029f50c913c40e0bd94fc9ba284
```

I think this should be and maybe followed by `go mod tidy`

```suggestion
	github.com/containers/storage v1.48.1-0.20230721123825-4a3a3019d765
```